frontend/src/services/graphQL.ts

```diff
@@ -27,14 +27,15 @@ export async function graphQLGetErrors(gqlData: any) {
 }
 
 export async function graphQLHandleError(error: AxiosError) {
-  const { auth, backend } = store.dispatch
+  const { auth, backend, ui } = store.dispatch
   console.error('GraphQL fetch error:', error, error?.response?.status)
   if (error?.response?.status === 401) {
-    const { auth } = store.dispatch
     auth.signInError('Session Expired')
   } else if (error?.response?.status === 403) {
     auth.checkSession()
     backend.set({ globalError: error.message })
+  } else if (error.message === 'Network Error') {
```

Comment:
why do you need this else if ?

Reply:
Just hiding the generic "Network Error" messages that appear when waking from sleep. The don't seem to have a response code. Just the message.